### PR TITLE
Refactor:extract common module postgresql-common

### PR DESCRIPTION
**Pull request by aderm**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

- Related issues

___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

**Comment by aderm** [comment]
yes, i want to add pg10

**Comment by wu-sheng** [comment]
Make sense.

**Comment by aderm** [comment]
Now pg10 and above are mainstream, and the meaning before maintenance is not big.

File: apm-sniffer/apm-sdk-plugin/postgresql-common/pom.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-deploy-plugin</artifactId>
+            </plugin>
+        </plugins>
+    </build>
```
Comment (kezhenxu94) [inline apm-sniffer/apm-sdk-plugin/postgresql-common/pom.xml]: Do we need this? What's this for?

Comment (aderm) [reply apm-sniffer/apm-sdk-plugin/postgresql-common/pom.xml]: it use for remote nexus repository. may be can delete for our project?

Comment (wu-sheng) [reply apm-sniffer/apm-sdk-plugin/postgresql-common/pom.xml]: We have the parent(root) pom file including the deploy to Apache repository. So, don't need this.

**Comment by aderm** [comment]
ok

**Pull request merged**